charts/sftp-server/values.yaml

```diff
@@ -156,6 +156,12 @@ probes:
   # -- Enable readiness probe
   readiness: true
 
+#
+# ref: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy
```

Comment:
Please use the `# -- ` prefix, so the comment will be part of the generated readme.
Please also wrap the ref into a markdown link as done in other comments (`([doc](<URL>`).

Reply:
Ok you are right